Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +10,232 @@ public static class BalanceStats
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0)
+            {
+                return "N/A.";
+            }
+
+            var HighestLossEver = decimal.MinValue;
+            var PersonWithHighestLoss = "";
+
+            for (int i = 0; i <= peopleAndBalances.Length - 1; i++)
+            {
+                var currentPersonData = peopleAndBalances[i].Split(',');
+                var allAmounts = currentPersonData[1..];
+
+                // calculateLoss
+
+                if (allAmounts.Length <= 1)
+                {
+                    return "N/A.";
+                }
+
+                var lossForCurrentPerson = 0M;
+
+                for (int j = 0; j < allAmounts.Length - 1; j++)
+                {
+                    var amount1 = decimal.Parse(allAmounts[j]);
+                    var amount2 = decimal.Parse(allAmounts[j + 1]);
+                    lossForCurrentPerson = amount1 - amount2;
+                }
+
+
+                //check if loss is greater then the current highest loss 
+                if (lossForCurrentPerson > HighestLossEver)
+                {
+                    HighestLossEver = lossForCurrentPerson;
+                    PersonWithHighestLoss = currentPersonData[0];
+                }
+
+            }
+
+            var answer = $"{PersonWithHighestLoss} lost the most money. -¤{HighestLossEver}.";
+
+            TextTable.Build(answer, 3);
+
+            return answer;
+
         }
 
         /// <summary>
         /// Return name and current money of the richest person.
         /// </summary>
         public static string FindRichestPerson(string[] peopleAndBalances)
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0)
+            {
+                return "N/A.";
+            }
+
+            var HighestBalance = decimal.MinValue;
+            var RichestPerson = new StringBuilder();
+
+            for (int i = 0; i <= peopleAndBalances.Length - 1; i++)
+            {
+                var currentPersonData = peopleAndBalances[i].Split(',');
+                var highestAmountOfPerson = decimal.Parse(currentPersonData[currentPersonData.Length - 1]);
+
+                if (highestAmountOfPerson > HighestBalance)
+                {
+                    HighestBalance = highestAmountOfPerson;
+                    var currentOne = RichestPerson.ToString();
+
+                    if (string.IsNullOrEmpty(currentOne))
+                    {
+                        RichestPerson.Append(currentPersonData[0]);
+                    }
+                    else
+                    {
+                        RichestPerson.Replace(currentOne, currentPersonData[0]);
+                    }
+
+
+                }
+                else if (highestAmountOfPerson == HighestBalance)
+                {
+                    if (i == peopleAndBalances.Length - 1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4937470)